libs/gltfio/src/FilamentAsset.cpp

```diff
@@ -38,8 +38,10 @@ FFilamentAsset::~FFilamentAsset() {
     releaseSourceData();
 
     // Destroy all instance objects. Instance entities / components are
-    // destroyed later in this method because they are owned by the asset.
+    // destroyed later in this method because they are owned by the asset
+    // (except for the root of the instance).
     for (FFilamentInstance* instance : mInstances) {
+        mEntityManager->destroy(instance->mRoot);
```

Comment:
Can't multiple instances have the same root?

Reply:
Each instance is created via a call to `createInstance`, so in that case, it seems no.

https://github.com/google/filament/blob/v1.31.2/libs/gltfio/src/AssetLoader.cpp#L488

Reply:
👍